ts/src/test/static/response/timex.json

```diff
@@ -48,11 +731,2491 @@
                         "amount": 0.005,
                         "cost": 37.80355,
                         "takerOrMaker": null,
-                        "fee": null
+                        "fee": {
+                            "cost": null,
+                            "currency": null
+                        },
+                        "fees": []
+                    },
+                    {
+                        "info": {
+                            "id": "498989",
```

Comment:
@ttodua I think we have way too many trades here making the test too long

Reply:
ah, havent noticed during push. cleaned now

Reply:
thanks!